lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1991,11 +1991,13 @@ LogicalResult FIRRTLLowering::visitDecl(RegOp op) {
     return setLowering(op, Value());
 
   // Add symbol if DontTouch annotation present.
+  auto symName = op.inner_symAttr();
```

Comment:
Can we lower don't touch to symbols much earlier in the pipeline and strip don't touch?

Reply:
could be a follow-up since this is large enough.

Reply:
Yeah that's a good thought. We could even do this as early as the `FIRParser`, when the ops gather their annotations before calling `builder.create<...>(...)`. That could be the opportunity to pass in a symbol name right from the beginning.

Reply:
Create #2102 to keep tabs on this.